tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html

```diff
@@ -84,9 +84,9 @@
+
 <div id="wp-tec-tickets-emails-footer-content-editor-container" class="wp-editor-container"><textarea class="wp-editor-area" style="height: 200px" cols="40" name="tec-tickets-emails-footer-content" id="tec-tickets-emails-footer-content"></textarea></div>
 </div>
 
-<label class="screen-reader-text">Add custom links and instructions to the bottom of your emails.</label><p class="tooltip description">Add custom links and instructions to the bottom of your emails.</p></div></fieldset><div class="clear"></div>
```

Comment:
pointiong out this `clear` getting removed

Reply:
Yes, we removed several of them - we don't use floats any more